### PR TITLE
fix(ui): v3 UI tweaks

### PR DESCRIPTION
**Pull request by alexec**
Signed-off-by: Alex Collins <alex_collins@intuit.com>

Checklist:

* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo/blob/master/USERS.md).

### CODE REVIEW

File: ui/src/app/workflows/components/workflow-creator.tsx [hunk 1]
```diff
@@ -93,9 +93,9 @@ export const WorkflowCreator = ({namespace, onCreate}: {namespace: string; onCre
                         entrypoints={(workflowTemplate.spec.templates || []).map(t => t.name)}
                         parameters={workflowTemplate.spec.arguments.parameters || []}
                     />
-                    <Button outline={true} onClick={() => setStage('full-editor')}>
-                        Edit using full workflow options
-                    </Button>
+                    <a onClick={() => setStage('full-editor')}>
```
Comment (alexec) [inline ui/src/app/workflows/components/workflow-creator.tsx:96]: image

File: ui/src/app/workflows/components/workflow-details/workflow-details.tsx [hunk 2]
```diff
@@ -176,7 +176,13 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
         const retryWatch = new RetryWatch<Workflow>(
             () => services.workflows.watch({name, namespace}),
             () => setError(null),
-            e => setWorkflow(e.object),
+            e => {
```
Comment (alexec) [inline ui/src/app/workflows/components/workflow-details/workflow-details.tsx:179]: When the workflow is deleted, we get a message and we still show the workflow itself:

image

File: ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx [hunk 3]
```diff
@@ -142,6 +142,11 @@ const WorkflowNodeSummary = (props: Props) => {
                         <i className='fa fa-bars' /> main logs
                     </DropDownButton>
                 )}{' '}
+                {props.node.type === 'Pod' && props.onShowEvents && (
```
Comment (alexec) [inline ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx:145]: Moved events next to logs:

image

File: ui/src/app/workflows/components/workflow-details/workflow-details.tsx [hunk 4]
```diff
@@ -263,6 +270,7 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
                     {parsedSidePanel.type === 'logs' && (
                         <WorkflowLogsViewer workflow={workflow} nodeId={parsedSidePanel.nodeId} container={parsedSidePanel.container} archived={false} />
                     )}
+                    {parsedSidePanel.type === 'events' && <EventsPanel namespace={namespace} kind='Pod' name={parsedSidePanel.nodeId} />}
```
Comment (alexec) [inline ui/src/app/workflows/components/workflow-details/workflow-details.tsx:273]: Node events now appear is a much more spacious side-panel:

image

File: ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx [hunk 5]
```diff
@@ -366,11 +377,6 @@ export const WorkflowNodeInfo = (props: Props) => (
                         </div>
                     )
                 },
-                props.node.type === 'Pod' && {
```
Comment (alexec) [inline ui/src/app/workflows/components/workflow-node-info/workflow-node-info.tsx:369]: removed this tab

**Comment by alexec** [comment]
e2e test failure unrelated to PR

File: ui/src/app/workflows/components/workflow-creator.tsx (hunk 1 above)
Comment (alexec) [reply ui/src/app/workflows/components/workflow-creator.tsx:96]: changed for consistency

**Review by rbreeze — APPROVED** [review]
LGTM

**Pull request merged**